guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Replace the text <strong>Initiatives </strong><em>(insert link to premium nb)</em> with a relative link to the url. The exact text should be:</p><p> </p><p>&gt; _Note:_ The pattern to add, search, get, update and delete <code>Sites</code> and <code>Pages</code> is identical to that shown in the [**Initiatives** guide](../enterprise-sites-guide)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
|**Before**|**After**|
|---|---|
![image](https://github.com/Esri/arcgis-python-api/assets/1399179/bf64d8f9-6ae0-406f-8de0-69ae0265ed04)|![image](https://github.com/Esri/arcgis-python-api/assets/1399179/ab388626-45d4-40a6-becb-49c989af0e1a)